t/assets/sass/themed-bootstrap.scss

```diff
@@ -0,0 +1,3 @@
+@import "../cache/raw.githubusercontent.com/twbs/bootstrap-sass/a73cc0f0e5c794206e9a70bc0b67e67cf37c1bca/assets/stylesheets/_bootstrap.scss";
+
+h1 { color: red; }
```

Comment:
You can jut put this in the `__DATA__` section of the unit test instead. Look at the other tests for examples.